### PR TITLE
BDOG-1619 adds better descriptions based on initiatives

### PR DESCRIPTION
**Pull request by TimothyFothergill**

### CODE REVIEW

File: app/uk/gov/hmrc/platforminitiatives/connectors/ServiceDependenciesConnector.scala [hunk 1]
```diff
@@ -46,9 +46,9 @@ class ServiceDependenciesConnector @Inject() (
   }
 
   def getServiceDependency(
-    group     : String,
-    artefact  : String,
-    range     : String = "[0.0.0,)"
+    group       : String,
+    artefact    : String,
+    range       : String = "[0.0.0,)",
```
Comment (beny23) [inline app/uk/gov/hmrc/platforminitiatives/connectors/ServiceDependenciesConnector.scala]: OBSERVATION: Is this dangling comma intentional?

File: app/uk/gov/hmrc/platforminitiatives/services/PlatformInitiativesService.scala [hunk 2]
```diff
@@ -34,33 +34,46 @@ class PlatformInitiativesService @Inject()(
   implicit val hc: HeaderCarrier = HeaderCarrier()
 
   def allPlatformInitiatives(team: Option[String] = None)(implicit ec: ExecutionContext): Future[Seq[PlatformInitiative]] = {
+    println(team)
```
Comment (beny23) [inline app/uk/gov/hmrc/platforminitiatives/services/PlatformInitiativesService.scala]: RED: I'd expect this is not meant to be in here...

File: app/uk/gov/hmrc/platforminitiatives/services/PlatformInitiativesService.scala [hunk 3]
```diff
@@ -34,33 +34,46 @@ class PlatformInitiativesService @Inject()(
...
     val initiatives = Seq(
       createDefaultBranchInitiative(
         initiativeName        = "Update Default Branch Terminology",
-        initiativeDescription = "To update default branch names - [Default branch tracker](https://catalogue.tax.service.gov.uk/defaultbranch).",
+        initiativeDescription = team match {
+          case None       => "To update default branch names - [Default Branch Tracker](https://catalogue.tax.service.gov.uk/defaultbranch?name=&teamNames=&defaultBranch=master) | [Confluence](https://confluence.tools.tax.service.gov.uk/display/TEC/2021/10/08/Default+Branch+Migration%3A+How+To)."
+          case Some(team) => s"To update default branch names - [Default Branch Tracker for $team](https://catalogue.tax.service.gov.uk/defaultbranch?name=&teamNames=$team&defaultBranch=master) | [Confluence](https://confluence.tools.tax.service.gov.uk/display/TEC/2021/10/08/Default+Branch+Migration%3A+How+To)."
```
Comment (beny23) [inline app/uk/gov/hmrc/platforminitiatives/services/PlatformInitiativesService.scala:42]: OBSERVATION: Could we put these into the messages resources?

Comment (TimothyFothergill) [reply app/uk/gov/hmrc/platforminitiatives/services/PlatformInitiativesService.scala]: Updated to remove the duplication of descriptions

**Pull request merged**